circle.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 jobs:
-  build_emscripten:
+  emscripten:
```

Comment:
Why rename these?

Reply:
I remember the long name not being fully visible in some parts of some UI.

Reply:
I really liked the long names, they were descriptive.